cmd/config.go

```diff
@@ -46,7 +47,8 @@ var configurationSetCmd = &cobra.Command{
 	Short: "Set a value in odo config file",
 	Long: `Set an individual value in the Odo configuration file 
 Available Parameters:
-UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+UpdateNotification - Controls if an update notification is shown or not (true or false)
+Timeout            - timeout(in seconds) for openshift server connection check(1 to 99)`,
 	Example: `
    # Set a configuration value
```

Comment:
It might be a good idea to add an example for the timeout too and separate them

Reply:
its also added